src/styles/blocks/calendar.scss

```
@@ -0,0 +1,59 @@
+@import "./utils/variables";
```

```suggestion
@import "./utils/variables"; 
```
Don't need to add these files. You did it in main.scss